src/routes/api/playlists/preloaded/+server.ts

```diff
@@ -0,0 +1,22 @@
+import { readPath, weaklyAuthenticate } from '$lib/server/firebaseUtils';
+import { json } from '@sveltejs/kit';
+import type { RequestHandler } from './$types';
+
+export const GET = (async (request) => {
+  const user = await weaklyAuthenticate(request);
```

Comment:
I don't think this is doing anything? `weaklyAuthenticate` won't throw an error if the auth fails, so we are just waiting on something for no reason. I see that it is the same way in the preloaded decks, that is a mistake! Quick win to speed things up by ~100ms would be to remove it from there (and here)

Reply:
I remember now, I think this is an artifact from when we were returning something else with preloaded (org decks?) so that they would sort first

Reply:
good point! but you're right i don't think we need it anymore, and i think the sorting is being handled in app separately. i'll remove and test to make sure nothing breaks